modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/portlet/action/GetContentDashboardItemInfoMVCResourceCommand.java

```diff
@@ -432,6 +470,54 @@ private JSONArray _getViewURLsJSONArray(
 			).toArray());
 	}
 
+	private boolean _isShareable(
```

Comment:
I don't know if we are checking that the item is a document or a blog in order to be shareable. Right now I'm able to see the share button in Web Content Articles and I shouldn't.